CHANGELOG.md

```diff
@@ -14,7 +14,7 @@
 
 ### Major
 
```

Comment:
Decided to touch Changelog to keep  log search functionality on a per a component bases

Reply:
Hm…for just this file, can we use something like "OverlayPanel (formerly Sheet)" or similar? There will likely be some stray references to Sheet, and it's also a name people are familiar with. (We really should have done this when Flyout became Popover — whoops)

Reply:
DONE